templates/essentials.yaml

```diff
@@ -74,7 +74,6 @@ Resources:
             Principal:
               AWS:
                 - !Sub 'arn:aws:iam::${AWS::AccountId}:root'
-                - !ImportValue us-east-1-bootstrap-CfServiceRoleArn
```

Comment:
i think whatever role that the CI is using for deployment still needs access otherwise the CI will not be able toupdatethis resource.  For Sage-Bionetworks-IT/organizations-infra it will be `!ImportValue us-east-1-sagebase-github-oidc-sage-bionetworks-it-ProviderRoleArn-organizationsinfra`

Reply:
Good catch! I thought that key was next to go...